Tools/ardupilotwaf/toolchain.py

```diff
@@ -16,7 +16,7 @@ def configure(cfg):
 from waflib.Configure import conf
 from waflib.Tools import compiler_c, compiler_cxx
 from waflib.Tools import clang, clangxx, gcc, gxx
-
+from waflib import Logs
 import os
```

Comment:
Please leave the empty line there, it's nice to separate library modules from "system" modules.